legal-api/src/legal_api/services/authz.py

```diff
@@ -469,12 +475,16 @@ def get_allowed_filings(business: Business,
     for allowable_filing_key, allowable_filing_value in allowable_filings.items():
         # skip if business does not exist and filing is not required
         # skip if this filing does not need to be returned for existing businesses
-        if bool(business) ^ allowable_filing_value.get('businessExists', True):
-            continue
+        # might re-active after update the amalgamation filing allowable method
+        # if bool(business) ^ allowable_filing_value.get('businessExists', True):
+        #     continue
```

Comment:
i don't think we want to remove this.  

I think if you do this then `incorporationApplication` and `registration` filings will show up in the allowable actions returned in the business response for an existing business

Reply:
I didn't remove it. I moved it here https://github.com/bcgov/lear/blob/b1482fee786a0d80e383b7ba7f1cefffd7875a2e/legal-api/src/legal_api/services/authz.py#L485-L487

If we keep that, we cannot process the amalgamation filing because `business` is `none` and `businessExists` is `True`